### PR TITLE
parameters and deploy addresses

### PR DESCRIPTION
**Pull request by hensha256**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "publishConfig": {
     "access": "public"
   },
-  "version": "1.1.1",
+  "version": "1.2.1",
```
Comment (ewilz) [inline package.json:8]: next version should be 1.2.0??

File: deploy-addresses/optimism-goerli.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "UniversalRouter": "0x4648a43B2C14Da09FdF82B161150d3F634f40491",
```
Comment (ewilz) [inline deploy-addresses/optimism-goerli.json:2]: these ended up being the same exact address despite different constructor params?

Comment (hensha256) [reply deploy-addresses/optimism-goerli.json:2]: yeah its deployed using CREATE not CREATE2

Comment (ewilz) [reply deploy-addresses/optimism-goerli.json:2]: ahh ok

**Review by ewilz — APPROVED** [review]

**Pull request merged**